packages/basics/test/json-parse.js

```diff
@@ -15,10 +15,21 @@ describe('JSONParse', () => {
                 done();
             });
     });
+    it('should return a value (yajs)', (done) => {
+        from(['{ "a": 1 }'])
+            .pipe(ezs('JSONParse', { legacy: true }))
```

Comment:
Should'nt it be `legacy: false`.
README says that default value is `true`.

Reply:
right, too